tests/unit/scripts/test_lint.py

```diff
@@ -88,6 +88,8 @@
     [[mail]]
         task event mail interval    = PT4M # deliberately added lots of spaces.
 
+CYLC_VERSION={{CYLC_VERSION}}
```

Comment:
We would expect to find this in fragments like:

```ini
[runtime]
    [[root]]
        [[[environment]]]
            CYLC_VERSION = {{ CYLC_VERSION }}
            ROSE_VERSION = {{ ROSE_VERSION }}
            FCM_VERSION = {{ FCM_VERSION }}
```

Reply:
Might as well put it in realistically.

Reply:
Especially since this revealed a fundamental problem with my original implementation: It was picking up _EVERY LINE_!